### DISCUSSION TITLE
Ability to report firmware as a later version?

### DISCUSSION THREAD

**Discussion by HeyItsDaddy** [gtxaspec/wz_mini_hacks#799] [Q&A] [answered]
I have a bunch of Cam V3's...and because I forgot to set the DISABLE_FW_UPGRADE="true" flag in the wz_mini.conf file, most of them got updated to the latest firmware, which I now have to reset each and roll them back to 4.36.9.139.  My question is, is there a way to report a later version of the firmware up to the Wyze App so that the app stops trying to force update them?

I still haven't found a great way to access the cameras remotely (would love an app that works on both Android an iOS), but saving off all of the streams using RTSP has been awesome.  The Wyze App is pretty convenient, though...but no longer since they don't allow you to see any camera on an older firmware.  The RTSP option is the MAIN reason I'm using these cameras, though, so not worth it to me doing the subscription service...I'd rather have no remote access than no RTSP stream saving.  So, if you also have any suggestions for remote viewing of my saved files and/or the RTSP streams themselves, please let me know.  I'm double-NAT'd, so almost nothing works well remotely...haven't been able to get Agent DVR, Blue Iris, or AnyCam to work reliably remotely.

**Comment by tschwei** [comment, marked as answer]
I also ran into this problem this week with my V3 cams (The Wyze app forces upgrading firmware so it's no longer compatible with RTSP).  So either we can't use mini_hacks anymore, or we can't use the Wyze app.  :-(

BUT! 
I noticed that there is a recent "add firmware spoofing" commit that spoofs the Wyze app to think the V3 cam firmware is running version 4.36.13.0416 instead of version 4.36.9.139!

So if you are using V3 camera with wz_mini_hacks, just download the latest master.zip and replace the wz_mini folder on your SD card with it, and poof! you can keep using the Wyze app!  :-)

So far, everything in the Wyze app seems to work fine with this version spoofing, and it allows everyone to keep using the Wyze app with RTSP!  Very cool!

Thanks to everyone for this awesome hack!

**Reply by HeyItsDaddy** [reply]
With the one VERY important added note of:
NOTE:  Make sure you set DISABLE_FW_UPGRADE="true" in your wz_mini.conf file to make this firmware spoof work!!!

Marking your reply as the answer!  Thanks you @tschwei and the developers that have made this tool amazing!!!

**Reply by gtxaspec** [reply]
i'll turn this on by default, good point

**Reply by HeyItsDaddy** [reply]
Hey, @gtxaspec , one day if you're bored, and this is a very low priority, might want to update places wz_mini_hacks reports FW version to add a (spoofed; actual is x.x.x.x) any place you report firmware.  For instance, in the diagnostics.cgi screen it awesomely now reports the spoofed version, but that's exactly the place where owners like us would want to know actual FW vs spoofed FW version.  So, just a thought for the future.

**Reply by robrahn123** [reply]
I was a little confused.  Basically the newest version of wz_mini_hacks has firmware spoofing built in unlike old versions.  So if you start from scratch you are all set.  Will have to update new wz_mini.conf file with your old settings.  Also update authorized_keys file if using WinSCP for remote access.  Working like a top as of now!  Thanks for the update!
